CONTRIBUTION.md

```diff
@@ -0,0 +1,103 @@
+# Contribute to DB4S
+
+Thanks for your interest in our project.  
+This document is intended to serve as lighthouse for contributors, not an executor.
+
+## Contents
+- [How to ask about DB4S](#how-to-ask-about-db4s)
+- [How to contribute?](#how-to-contribute)
+    - [Report bug](#report-bug)
+    - [Suggest enhancement](#suggest-enhancement)
+    - [Translation](#translation)
+- [Styleguide](#styleguide)
+    - [CPP source code](#cpp-source-code)
+    - [Git commit message](#git-commit-message)
+
+## How to ask about DB4S?
+If you have any questions about our project, please check the following first.
+
+- Look up related topics on the [wiki](https://github.com/sqlitebrowser/sqlitebrowser/wiki)
+- Search for existing similar issues
+
+If your question is still not resolved, you can create an issue and ask a question.  
+A template from the GitHub issue will help you write a good bug report.
```

Comment:
We were talking about asking questions, so the reference to "good bug report" seems inconsistent. Maybe this should say a "good question".

Reply:
Nice catch, I just modified it.